great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -29,30 +29,75 @@ class PandasDatasourceError(Exception):
     pass
 
 
-_ASSET_MODELS = _generate_pandas_data_asset_models(
-    _FilesystemDataAsset,
-    whitelist=(
-        "read_csv",
-        "read_json",
-        "read_excel",
-        "read_parquet",
-    ),
+_BLACK_LIST = (
+    # "read_csv",
+    # "read_json",
+    # "read_excel",
+    # "read_parquet",
+    # "read_clipboard",
+    # "read_feather",
+    "read_fwf",  # unhandled type
+    # "read_gbq",
+    # "read_hdf",
+    # "read_html",
+    # "read_orc",
+    # "read_pickle",
+    # "read_sas",
+    # "read_spss",
+    "read_sql",  # type-name conflict
+    # "read_sql_query",
+    # "read_sql_table",
+    "read_table",  # type-name conflict
+    # "read_xml",
 )
 
+_ASSET_MODELS = _generate_pandas_data_asset_models(
+    _FilesystemDataAsset, blacklist=_BLACK_LIST
+)
 
+ClipboardAsset = _ASSET_MODELS["clipboard"]
 CSVAsset = _ASSET_MODELS["csv"]
 ExcelAsset = _ASSET_MODELS["excel"]
+FeatherAsset = _ASSET_MODELS["feather"]
+GBQAsset = _ASSET_MODELS["gbq"]
+HDFAsset = _ASSET_MODELS["hdf"]
+HTMLAsset = _ASSET_MODELS["html"]
 JSONAsset = _ASSET_MODELS["json"]
+ORCAsset = _ASSET_MODELS["orc"]
```

Comment:
😢 unfortunately `ORC` is an acronym, so no `OrcAsset` 😆

Reply:
🤣